docs/api/ShallowWrapper/contains.md

```diff
@@ -56,6 +56,17 @@ expect(wrapper.contains([
 ])).to.equal(false);
 ```
 
+```jsx
+let calculatedValue = 2 + 2;
```

Comment:
this needs to be a `const`

Reply:
Sorry I missed this, I've fixed this, and rebased to 1 commit on the latest master